src/ResourceManagement/HDInsightJob/HDInsightJob.Tests/TestUtils.cs

```diff
@@ -23,25 +23,37 @@ namespace HDInsightJob.Tests
 {
     public static class TestUtils
     {
-        public static string ClusterName = "pattipakalinux330.azurehdinsight.net";
+        // ADL clusters are supported only for Linux. This flag is to configure ADL clusters.
+        public static bool AdlCluster = true;
+        public static string AdlAccountName = "pattipakaadls2";
+
+        // ADL store access details like default storage root, application Id and certificate details to access storage
+        public static string DefaultStorageRoot = "/clusters/pattipakaadls";
+        public static string ApplicationId = "";
+        public static string AdlCertificatePath = @"D:\ADLS\pattipakalinuxadls.pfx";
```

Comment:
These are user input to submit jobs to an existing ADL cluster. These inputs are required to run jobs on real cluster and then retrieve job outputs from datalake store. I have included these details as an example such that other users who are trying to run these tests which would these appropriately to run jobs on real cluster.